tool/tsh/common/tsh.go

```diff
@@ -1122,6 +1122,9 @@ func Run(ctx context.Context, args []string, opts ...CliOption) error {
 	// MFA subcommands.
 	mfa := newMFACommand(app)
 
+	flatten := app.Command("flatten", "Flattens an identity file into a profile stored in ~/.tsh or TELEPORT_HOME.")
```

Comment:
@hugoShaka @Joerger I think the functionality itself makes sense but instead of introducing yet another command I think it would make more sense to teach `tsh login -i <identity>` support this use-case.